pkg/virt-launcher/monitor.go

```diff
@@ -229,34 +228,16 @@ func pidExists(pid int) (exists bool, isZombie bool, err error) {
 	return exists, isZombie, nil
 }
 
-func FindPid(commandNamePrefix string) (int, error) {
-	entries, err := filepath.Glob("/proc/*/cmdline")
+func FindPid(domainName string) (int, error) {
+	content, err := ioutil.ReadFile("/run/libvirt/qemu/" + domainName + ".pid")
```

Comment:
I was looking at the PR https://github.com/kubevirt/kubevirt/pull/7631 locally and the location of the PID file appears to be different depending on root/non-root mode:
```
bash-4.4# ls -la /run/libvirt/qemu/default_vmi-fedora.pid
-rw-r--r--. 1 root root 2 Apr 27 07:24 /run/libvirt/qemu/default_vmi-fedora.pid
```

```
bash-4.4$ ls -la /run/libvirt/qemu/default_vmi-fedora.pid
ls: cannot access '/run/libvirt/qemu/default_vmi-fedora.pid': No such file or directory
bash-4.4$ ls -la /run/libvirt/qemu/run/default_vmi-fedora.pid 
-rw-r--r--. 1 qemu qemu 3 Apr 27 07:27 /run/libvirt/qemu/run/default_vmi-fedora.pid
```

Reply:
Very good point, fixed. It's actually the same path with and without the PR you're mentioning, so we should be good to go there.